build.gradle

```diff
@@ -20,7 +20,8 @@ ext {
 
   // Third parties
   offheapVersion = '2.2.2'
-  statisticVersion = '1.1.0'
+  managementVersion = '5.0.0.beta'
```

Comment:
Yes it was, and it was in code earlier. It changed at some point and you probably kept it around while resolving a conflict. Like I said, needs to go. The same prop is defined correctly a few lines below.